assets/src/edit-story/elements/image/display.js

```diff
@@ -38,6 +40,24 @@ function ImageDisplay({ element, box }) {
   const { width, height } = box;
   const ref = useRef();
 
+  let initialSrcType = 'smallest';
+  let initialSrc =
+    resource.sizes?.['web-stories-thumbnail']?.source_url ||
+    resource.sizes?.medium?.source_url;
```

Comment:
Unsplash images don't have any of these 2. If you open a story that has an Unsplash image, it wouldn't be cached yet and these 2 would not exist. Wouldn't that result in no image being displayed?

Can't you instead simply select the src with the smallest width in resource.sizes that has the same aspect ratio as the original image?

Reply:
Alright, for some reason 3PMedia uses
```js
['web_stories_thumbnail', sizesFromBiggest[sizesFromBiggest.length - 1]],
```
contrary to
```js
const STORY_THUMBNAIL_IMAGE_SIZE = 'web-stories-thumbnail';
```

---
  
> Wouldn't that result in no image being displayed?

No, in the worst case you will just wait for the fullsize image (like now).